main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"runtime"
 	"strings"
 
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
```

Comment:
The import order seems odd.

Reply:
good call, thanks. just pushed a change to resolve.